File: EasyPlan.Web/Components/Mapper/MarkMapper.cs
using EasyPlan.DomainModel.Entities;

namespace EasyPlan.Web.Components.Mapper
{
    public static class MarkMapper
    {
        public static object Map(Mark entity)
        {
            return new
            {
                id = entity.Id,
                criterionId = entity.CriterionId,
                itemId = entity.ItemId,
                value = entity.Value
            };
        }
    }
}

File: EasyPlan.Web/App/mappers/markMapper.js
define(['models/mark'], function (Mark) {
    return {
        map: map,
    }

    function map(src) {
        return new Mark({
            id: src.id,
            value: src.value,
            criterionId: src.criterionId,
            itemId: src.itemId
        })
    }
});